config/environments/development.rb

```diff
@@ -14,8 +14,23 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = { host:'localhost', port: '3000' }
```

Comment:
This is a duplicate of line 18 and will override it. So you can delete the previous one.